libraries/ms-helper/test/test_heartbeat_rx.c

```diff
@@ -91,13 +93,13 @@ void test_heartbeat_rx(void) {
     .expected_bitset = CAN_ACK_EXPECTED_DEVICES(TEST_HEARTBEAT_RX_CAN_DEVICE_ID),
   };
 
-  CAN_TRANSMIT_POWERTRAIN_HEARTBEAT(&req);
+  CAN_TRANSMIT_BPS_HEARTBEAT(&req,TEST_HEARTBEAT_STATUS);
   MS_TEST_HELPER_CAN_TX_RX_WITH_ACK(TEST_HEARTBEAT_RX_RX_CAN_TX, TEST_HEARTBEAT_RX_RX_CAN_RX);
   TEST_ASSERT_TRUE(context.executed);
 
   expected_status = CAN_ACK_STATUS_INVALID;
   context.ret_code = false;
-  CAN_TRANSMIT_POWERTRAIN_HEARTBEAT(&req);
+  CAN_TRANSMIT_BPS_HEARTBEAT(&req,TEST_HEARTBEAT_STATUS);
   MS_TEST_HELPER_CAN_TX_RX_WITH_ACK(TEST_HEARTBEAT_RX_RX_CAN_TX, TEST_HEARTBEAT_RX_RX_CAN_RX);
   TEST_ASSERT_TRUE(context.executed);
-}
+}
```

Comment:
I think format will fail here, please run `make format` (and if it doesn't add it, add a newline at the end of this file).